src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/weapons/ExplosiveBow.java

```diff
@@ -30,14 +31,7 @@
  */
 public class ExplosiveBow extends SlimefunBow {
 
-    private final ItemSetting<Integer> range = new ItemSetting<Integer>("explosion-range", 3) {
-
-        @Override
-        public boolean validateInput(Integer input) {
-            return super.validateInput(input) && input > 0;
-        }
-
-    };
+    private final ItemSetting<Integer> range = new IntRangeSetting("explosion-range", 1, 3, Integer.MAX_VALUE);
```

Comment:
Integer.MAX_VALUE should not be used here. We don't want some crazy server owner to make an explosive bow that launches players at the other side of the world.

Reply:
Their own fault tbh.

Reply:
> Their own fault tbh.

Well get ready for bug reports of "explosive bow blows from 100k blocks away" then. Lol

Reply:
That would not be a bug though, if that happens then that's very much inline with what they configured.
If you throw a brick at a window, don't be surprised the window breaks...